core/ecsql/common/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "@itwin/ecsql-common",
+  "version": "4.2.0-dev.1",
+  "description": "ECSql component that can be reference on backend and frontend",
+  "main": "lib/cjs/ecsql-common.js",
+  "module": "lib/esm/ecsql-common.js",
+  "typings": "lib/cjs/ecsql-common",
+  "license": "MIT",
+  "scripts": {
+    "build": "npm run -s build:cjs && npm run -s build:esm",
+    "build:cjs": "tsc 1>&2 --outDir lib/cjs",
+    "build:esm": "tsc 1>&2 --module ES2020 --outDir lib/esm",
+    "clean": "rimraf lib .rush/temp/package-deps*.json",
+    "docs": "betools docs --includes=../../generated-docs/extract --json=../../generated-docs/core/ecsql-common/file.json --tsIndexFile=./ecsql-common.ts --onlyJson",
+    "extract-api": "betools extract-api --entry=ecsql-common",
+    "lint": "eslint -f visualstudio \"./src/**/*.ts\" 1>&2",
+    "test": "mocha",
+    "cover": "nyc npm -s test"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/iTwin/itwinjs-core.git",
+    "directory": "core/ecsql/common"
+  },
+  "keywords": [
+    "Bentley",
+    "BIM",
+    "iModel",
+    "digital-twin",
+    "iTwin",
+    "ecsql"
+  ],
+  "author": {
+    "name": "Bentley Systems, Inc.",
+    "url": "http://www.bentley.com"
+  },
+  "dependencies": {
+    "flatbuffers": "~1.12.0",
+    "js-base64": "^3.6.1"
+  },
+  "peerDependencies": {
+    "@itwin/core-bentley": "workspace:^4.2.0-dev.1"
```

Comment:
i believe the current usage of core-bentley is pure/safe, so doesnt need to be a dev+peer, can be a direct dep